app/controllers/profile_controller.rb

```diff
@@ -8,4 +8,7 @@ def show
     end
   end
   
+  def register
```

Comment:
En csinalnek egy uj controllert ennek: `RegistrationsController` vagy hasonlo es ott lennenek a szokasos RESTful actionok. Es ha valamikor veletlen bonyolultabb regisztracios folyamat lesz es esetleg sajat modellt is kap a regisztracio (ahogy most is van) akkor nem lesz gond, hogy kinonetek a controllert. A controllerekkel amugy sem erdemes sporolni.